tls/s2n_cipher_suites.c

```diff
@@ -1157,7 +1157,7 @@ int s2n_set_cipher_as_client(struct s2n_connection *conn, uint8_t wire[S2N_TLS_C
 
 static int s2n_wire_ciphers_contain(const uint8_t *match, const uint8_t *wire, uint32_t count, uint32_t cipher_suite_len)
 {
-    for (uint32_t i = 0; i < count; i++) {
+    for (size_t i = 0; i < (size_t) count; i++) {
```

Comment:
Doesn't the sign-compare check only apply if the signs are different? It doesn't seem like this cast is needed, but I could be wrong. If there is a reason to make this cast, wouldn't a lot of the other for loops also have to be updated?